src/components/redux/books/books.js

```diff
@@ -1,17 +1,48 @@
+import {
```

Comment:
- [ OPTIONAL] The redux folder should be removed from the components folder and put directly in the src folder as it is not a component. Kindly fix this before merging the branch.